roles/base/tasks/main.yml

```diff
@@ -18,6 +39,7 @@
     - librbd-dev
     - lshw
   when: ansible_os_family == "Debian"
+  ignore_errors: yes
```

Comment:
I don't like to ignore errors. Unfortunately, this task intermittently returns apt failures. If anyone knows a fix/workaround for that, we can add that and remove the ignore errors.